recipe/meta.yaml

```diff
@@ -24,16 +24,24 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - toml
+    - tomli >=1.1.0
 
 test:
   requires:
     - pip
+    - coverage ==7.0.5
+    - pint
+    - pytest ==7.2.1
+    - pytest-cov ==4.0.0
   imports:
     - vulture
+  source_files:
+    - vulture
+    - tests
   commands:
     - pip check
     - vulture --help
+    - pytest --cov vulture --cov-report=html --cov-report=term --cov-report=xml --cov-append
```

Comment:
If we want to run the test suite, that is ok

If we find this takes a while, we might run some subset

Mostly what we are trying to catch here is packaging issues. Generally we think upstream projects are better positioned to run comprehensive CI

Reply:
That's fair. Given that the tests run from start to finish in around 10 seconds, I figured it was fine.